src/useq/_mda_sequence.py

```diff
@@ -484,15 +495,23 @@ def iter_sequence(
                     base_event_kwargs=event_kwargs.copy(),
                     event_kwarg_overrides=pos_overrides,
                     position_offsets=_offsets,
+                    shutter_plan=(
+                        position.sequence.shutter_plan or sequence.shutter_plan
+                    ),
                 )
                 continue
             # note that position.sequence may be Falsey even if not None, for example
             # if all it has is an autofocus plan.  In that case, we don't recurse.
             # and we don't hit the continue statement, but we can use the autofocus plan
             elif position.sequence is not None and position.sequence.autofocus_plan:
                 autofocus_plan = position.sequence.autofocus_plan
+                shutter_plan = position.sequence.shutter_plan or sequence.shutter_plan
```

Comment:
_this_ should be the only thing you need. no?  we didn't need all the other stuff for autofocus plan, so I'm confused why we would need it for shutter plan